src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,34 @@
+.card__review-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars--4 {
+  display: flex;
+}
+    
+.card__review-count {
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Do not forget remove margin from last child. 
Solution shoud be same as on stars task